installer/release.sh

```diff
@@ -111,19 +104,25 @@ case "$displayver" in
 esac
 
 # Evaluate architecture
-ARCH="$(uname -m)"
-
-case "$ARCH" in
-i686)
```

Comment:
I would prefer the change from `uname -m` to `$MSYSTEM` to be done in a separate commit that still only handles i686 and x86_64, and add the aarch64 part in a separate commit.

Please let me know if you want assistance with that.